docs/develop/itp-and-third-party-cookies.md

```diff
@@ -14,7 +14,9 @@ If your Office Add-in and website must rely on third-party cookies, use the foll
 1. Set up [OAuth 2.0 Authorization](https://tools.ietf.org/html/rfc6749) so that the authenticating domain (in your case, the third-party that expects cookies) forwards an authorization token to your website. Use the token to establish a first-party session with a server-set Secure and [HttpOnly cookie](https://developer.mozilla.org/docs/Web/HTTP/Cookies#Secure_and_HttpOnly_cookies).
 1. Use the [Storage Access API](https://webkit.org/blog/8124/introducing-storage-access-api/) so that the third-party can request permission to get access to its first-party cookies. Current versions of Office on Mac and Office on the web both support this API.
     > [!NOTE]
-    > If you're using cookies for purposes other than authentication, then consider using `localStorage` for your scenario.
+    > If you're using cookies for purposes other than authentication, consider using `localStorage` for your scenario.
+    >
+    > However, note that starting in Version 115 of Chromium-based browsers, such as Chrome and Edge, [storage partitioning](https://developer.chrome.com/docs/privacy-sandbox/storage-partitioning/) is being tested to prevent specific side-channel cross-site tracking. This means that data stored by storage APIs, such as local storage, are only available to contexts with the same origin and the same top-level site. To work around this, in your browser, go to **chrome://flags** or **edge://flags**, then set the **Experimental third-party storage partitioning (#third-party-storage-partitioning)** flag to **Disabled**.
```

Comment:
Would end users have to do this too to use the add-in? If so, maybe local storage just isn't feasible for add-ins unless the dev knows that all the end users will not be using a Chromium-based browser.

Reply:
Yes. The team is discussing long-term solutions, so we'll likely update this article again. But we'll start with this note for now.